docs/src/rules/max-statements.md

```diff
@@ -88,43 +88,43 @@ Examples of **correct** code for this rule with the default `{ "max": 10 }` opti
 /*eslint max-statements: ["error", 10]*/
 
 function foo() {
-  var foo1 = 1;
-  var foo2 = 2;
-  var foo3 = 3;
-  var foo4 = 4;
-  var foo5 = 5;
-  var foo6 = 6;
-  var foo7 = 7;
-  var foo8 = 8;
-  var foo9 = 9;
+  const foo1 = 1;
+  const foo2 = 2;
+  const foo3 = 3;
+  const foo4 = 4;
+  const foo5 = 5;
+  const foo6 = 6;
+  const foo7 = 7;
+  const foo8 = 8;
+  const foo9 = 9;
   return function () { // 10
 
     // The number of statements in the inner function does not count toward the
     // statement maximum.
 
-    var bar;
-    var baz;
+    const bar;
+    const baz;
```

Comment:
```suggestion
    let bar;
    let baz;
```
when we use const to declare a variable, we must assign it a value at the time of declaration. so lets use `let` here.